pilot/pkg/bootstrap/server.go

```diff
@@ -199,6 +199,24 @@ func NewServer(args *PilotArgs, initFuncs ...func(*Server)) (*Server, error) {
 		internalStop:            make(chan struct{}),
 		istiodCertBundleWatcher: keycertbundle.NewWatcher(),
 	}
+
+	// Used for readiness, monitoring and debug handlers.
+	var (
+		whMu sync.RWMutex
+		wh   *inject.Webhook
+	)
+	whc := func() map[string]string {
+		whMu.RLock()
+		defer whMu.RUnlock()
+		if wh != nil {
+			return wh.Config.RawTemplates
+		}
+		return map[string]string{}
+	}
+	if err := s.initIstiodAdminServer(args, whc); err != nil {
```

Comment:
do we know that we won't get crashes, etc from all the endpoints? For example kubeClient being nil may be problematic ?

Might be worth testing that things work or fail gracefully if we were to sleep(infinity) right after this